### PR TITLE
fix: update output of `convert-k8s` command

### PR DESCRIPTION
**Pull request by smira**
This includes Sean's comments from #3278 and introduces a new flag which
is referenced in manual conversion process document.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

File: website/content/docs/v0.9/Guides/converting-control-plane.md [hunk 1]
```diff
@@ -39,7 +39,8 @@ Talos generated control plane static pod definitions and bootstrap manifests, pl
     talosctl -n <master node IP> get StaticPods.kubernetes.talos.dev
     talosctl -n <master node IP> get Manifests.kubernetes.talos.dev
 
-bootstrap manifests will only be applied for missing resources, existing resources will not be updated
+in order to remove self-hosted control plane, pod-checkpointer component needs to be disabled
+once pod-checkpointer is disabled, cluster shouldn't be rebooted until the entire conversion process is complete
```
Comment (andrewrynhard) [inline website/content/docs/v0.9/Guides/converting-control-plane.md:43]: nit: `once pod-checkpointer is disabled, cluster` -> `once pod-checkpointer is disabled, the cluster`
                                                                                                                                                  ^ HERE

(And similar edits in other places)

**Review by andrewrynhard — APPROVED** [review]

File: website/content/docs/v0.9/Guides/converting-control-plane.md (hunk 1 above)
Comment (smira) [reply website/content/docs/v0.9/Guides/converting-control-plane.md]: thanks, fixed!

**Review by andrewrynhard — APPROVED** [review]

**Pull request merged**